packages/console/src/components/MultiTextInput/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as textButtonStyles from '@/components/TextButton/index.module.scss';
+import Minus from '@/icons/Minus';
+
+import IconButton from '../IconButton';
+import TextInput from '../TextInput';
+import * as styles from './index.module.scss';
+import { MultiTextInputError } from './types';
+
+type Props = {
+  value: string[];
```

Comment:
`value` and `onChange` can be optional

Reply:
Refer to the comment by @simeng-li, maybe it's good to keep these required.
https://github.com/logto-io/logto/pull/472#discussion_r839397096

Reply:
OK, not a blocker.